### PR TITLE
Add ListPrivateSkillService

### LINKED ISSUE #1311: Add ListPrivateSkillService
**Actual Behaviour**
Not present. Required for the issue https://github.com/fossasia/susi.ai/issues/2551

**Expected Behaviour**
Shall return a list of private skills. 

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by harsh-jindal**
Fixes #1311 

Changes: getSkillList.json now returns lists of bots.

Screenshots for the change: 
Screenshot 2019-07-20 at 7 19 59 AM
`With search="test_bot"`
Screenshot 2019-07-20 at 7 20 11 AM

### CODE REVIEW

**Comment by harsh-jindal** [comment]
@akshatnitd @fragm3 @mariobehling Please review

**Comment by harsh-jindal** [comment]
@akshatnitd @fragm3 Done. Please review.

File: src/ai/susi/server/api/cms/ListPrivateSkillService.java [hunk 1]
```diff
@@ -0,0 +1,146 @@
+package ai.susi.server.api.cms;
+
+/**
+* ListPrivateSkillService
+* Copyright by Harsh Jindal, @harsh-jindal
+*
+* This library is free software; you can redistribute it and/or
+* modify it under the terms of the GNU Lesser General Public
```
Comment (fragm3) [inline src/ai/susi/server/api/cms/ListPrivateSkillService.java]: Remove all copyright

File: src/ai/susi/server/api/cms/ListPrivateSkillService.java [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+            result.put("chatbots", botObject);
+            result.put("accepted", true);
+            result.put("message", "Success: Fetched all Private Skills");
+            return new ServiceResponse(result);
+        } catch (Exception e) {
+            throw new APIException(500, "Failed to fetch the requested list!");
+        }
+
+    }
+}
```
Comment (fragm3) [inline src/ai/susi/server/api/cms/ListPrivateSkillService.java]: New line

**Comment by harsh-jindal** [comment]
@fragm3 Made the changes

**Pull request merged**